File: favdice.c
#include<stdio.h>
int main()
{
	int n,t,i = 0,j;
	double ans;
	scanf("%d",&t);
	while(i < t)
	{
		ans = 0;
		i ++;
		scanf("%d",&n);
		for(j = 1; j <=n;j++)
		{
			ans += n/(1.0*j);
		}
		printf("%.2lf\n",ans);
	}
	return 0;
}

File: favdice.py
def main():
	t = input()
	i = 0
	while (i < t):
		i += 1
		ans = 0
		n = input()
		for k in range(1,n+1):
			ans += n/(1.0*k)
		print "%.2f" % (ans)

if __name__ == '__main__':
	main()